### ISSUE TITLE
Controllers seem to be unresponsive when initiating scene

### ISSUE THREAD

**Issue by yuhyuhyuhD** [ManlyMarco/HS2-HF_Patch#5]
Hello when I try to start any action I, in the past, simply moved the controllers stick up or down to start or stop. A little while ago that stopped working so I uninstalled game and patch and redownloaded and installed everything however they still seem to not work. Is there something I can do to fix this issue?

**Comment by ManlyMarco** [comment]
This is in the VR module? You can try running without mods by removing winhttp.dll to see if that's the cause. Check output_log.txt in game directory for any exceptions or errors around the time when you try to do the action.

**Comment by yuhyuhyuhD** [comment]
Yes sorry this is with the VR module and with the winhttp.dll removed it still does not seem to work. I tried looking in the output_log file but to be honest im not really sure where to look it was very long. The controllers work fine to select things and open menu with and without mods its just the initiation that it cant do for some reason

**Comment by ManlyMarco** [comment]
Try grabbing latest vr module from http://www.illusion.jp/preview/honey2/download_vr.php
If that doesn't help try an older version. I don't have it on hand, it should be somewhere on the IS discord server.

**Comment by yuhyuhyuhD** [comment]
ok ill give it a try thank you for the help ik your probably busy af

**Comment by yuhyuhyuhD** [comment]
ok it wont let me download the newest vr module because the site is taking too long to respond do you have an invite to the IS server? or know where i can get one?

**Comment by ManlyMarco** [comment]
It's in the patch readme and in the release post on patreon.

**Comment by ManlyMarco** [comment]
The server is open to everyone. You don't need any privileges, only need to pass the bot verification. https://discord.gg/illusionsoft

**Comment by 15129509714** [comment]
插件全是英文怎么调中文啊？
